src/main/java/tests/OperationHandlersTest.java

```diff
@@ -0,0 +1,65 @@
+package tests;
+
+import core.basesyntax.*;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class OperationHandlersTest {
```

Comment:
The class name `OperationHandlersTest` should be more specific to reflect what is actually being tested. Consider naming the test class to match the class being tested, such as `BalanceOperationTest`, `SupplyOperationTest`, etc.